ui/app/components/secret-version-menu.js

```diff
@@ -1,60 +1,5 @@
-import { maybeQueryRecord } from 'vault/macros/maybe-query-record';
-import Component from '@ember/component';
-import { inject as service } from '@ember/service';
-import { alias, or } from '@ember/object/computed';
+import Component from '@glimmer/component';
 
```

Comment:
secret-version-menu used to house all the deletion actions, now it only handles version things.